src/mono/mono/mini/debugger-agent.c

```diff
@@ -8511,8 +8510,6 @@ method_commands_internal (int command, MonoMethod *method, MonoDomain *domain, g
 			mono_error_cleanup (error);
 			return ERR_INVALID_ARGUMENT;
 		}
-		if (!mono_verifier_is_method_valid_generic_instantiation (inflated))
```

Comment:
This, and probably some other places check whenever what the app wants to construct is valid, it shouldn't be removed.

Reply:
It always returns TRUE in netcore (with verified disabled)

Reply:
no I'm wrong.  Need to rework this.